### PR TITLE
Some typo and a little improvement

### PR DESCRIPTION
**Pull request by bySabi**
The content is very good. Thanks for make the hard work.

### CODE REVIEW

File: 03-specificity/README.md [hunk 1]
```diff
@@ -370,7 +370,7 @@ Not very useful for CSS-in-JS solutions.
 
 ### Emotion (Styled Components)
 
-Emotion generates CSS class names, we must care about source order and specificity as if we were writing regular CSS.
+Emotion generate CSS class names, we must care about source order and specificity as if we were writing regular CSS.
```
Comment (aarongarciah) [inline 03-specificity/README.md]: I think `Emotion generates` is correct 🤔

File: 03-specificity/README.md [hunk 2]
```diff
@@ -105,9 +105,9 @@ li a {
 }
 ```
 
-### `:not` pseudo-class
+### `:not` the negation *pseudo-class*
```
Comment (aarongarciah) [inline 03-specificity/README.md]: Why pseudo-class italic?

**Review by aarongarciah — COMMENTED** [review]
Thanks! Some minor comments.

File: 03-specificity/README.md (hunk 1 above)
Comment (bySabi) [reply 03-specificity/README.md]: I think the word `generates` does not exist.

File: 03-specificity/README.md (hunk 2 above)
Comment (bySabi) [reply 03-specificity/README.md]: To highlight it a bit with respect to `the negation` noun. But feel free to put it as you think is better, you are the UX man ;-)

File: 03-specificity/README.md (hunk 1 above)
Comment (aarongarciah) [reply 03-specificity/README.md]: It exists! [see 🤓](https://translate.google.com/#view=home&op=translate&sl=es&tl=en&text=esto%20genera%20nombres%20de%20clases%20CSS)

File: 03-specificity/README.md (hunk 2 above)
Comment (aarongarciah) [reply 03-specificity/README.md]: I prefer not ot use italics here 😄

**Comment by bySabi** [comment]
Your right with `generates` word  :-)

I hope is right now.

**Review by aarongarciah — APPROVED** [review]

**Comment by aarongarciah** [comment]
Thanks!

**Pull request merged**